### ISSUE TITLE
Bonds in allyl cation are not conjugated

### ISSUE THREAD

**Issue by greglandrum** [rdkit/rdkit#539]
Reported by Paolo Tosco
```python
In [2]: m = Chem.MolFromSmiles('C=C-[CH2+]')

In [3]: for b in m.GetBonds():
    print(b.GetIsConjugated())
   ...:     
False
False

In [4]: for a in m.GetAtoms():
    print(a.GetHybridization())
   ...:     
SP2
SP2
SP2
```

**Comment by ptosco** [comment]
I noticed that also aromatic carbocations such as cyclopropenyl and tropylium are not perceived as aromatic, nor can be kekulized. I have submitted a pull request https://github.com/rdkit/rdkit/pull/540 which should fix all of these.

**Comment by greglandrum** [comment]
Fixed by #540
